Documentation/ceph-encryption.md

```diff
@@ -0,0 +1,225 @@
+#  Ceph encryption at rest
```

Comment:
Since this is a design doc, let's actually move it under `/design/ceph`. The user documentation is only in the Documentation folder. After the feature is implemented, then we expect to update the Documentation folder, thanks!

Reply:
Will do.